packages/superset-ui-preset-chart-xy/src/components/tooltip/TooltipTable.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { CSSProperties, PureComponent } from 'react';
+import React, { CSSProperties, PureComponent, ReactNode } from 'react';
 
 type Props = {
   className?: string;
   data: {
-    key: string;
+    key: string | number;
+    keyColumn: ReactNode;
     keyStyle?: CSSProperties;
-    value: string | number;
+    valueColumn: ReactNode;
```

Comment:
are there cases where this is **not** `string/number/null`? `ReactNode` seems strange to have inside an array of data objects.

Reply:
I'm trying to be flexible in case extra formatting is needed or there are unforeseen thing we want to put in the value column. Also to make it same with `keyColumn`